### DISCUSSION TITLE
Thanks for helping make RedwoodJS amazing!

### DISCUSSION THREAD

**Discussion by ajcwebdev** [tailwindlabs/tailwindcss#3206] [Kind words]
We make sure RedwoodJS users can integrate any CSS library they want, but I know many users and members of the core team have gotten a ton of value from [integrating TailwindCSS](https://redwoodjs.com/docs/webpack-configuration#adding-tailwind-css).

We've been able to stand on the shoulders of giants with TailwindCSS and I know we're not the only one (hope your shoulders aren't too tired, Adam and Co.). Keep doing all the awesome stuff you do!

**Comment by adamwathan** [comment]
Thanks so much! I was super excited when I saw the Redwood website was built with Tailwind, really excited about the work y'all are doing over there 🤩
